proto/ligato/vpp/interfaces/interface.proto

```diff
@@ -257,30 +257,41 @@ message TapLink {
 
 // IPSecLink defines configuration for interface type: IPSEC_TUNNEL
 message IPSecLink {
+    enum Mode {
+        // point-to-point tunnel
+        POINT_TO_POINT = 0;
+        // point-to multipoint tunnel (supported starting from VPP 20.05)
+        POINT_TO_MULTIPOINT = 1;
+    };
+    // Mode of the IPIP tunnel
+    Mode tunnel_mode = 1;
+
+    // In VPP 21.06 and newer, IPSecLink serves just for creation of the link and thus all of the following
```

Comment:
Please move this comment to the `IPSecLink` message comment.

Reply:
done